documentation/docs/20-core-concepts/10-routing.md

```diff
@@ -12,6 +12,19 @@ At the heart of SvelteKit is a _filesystem-based router_. The routes of your app
 
 Each route directory contains one or more _route files_, which can be identified by their `+` prefix.
 
+Summary of the routing files:
+
+| File name         | Role                                                                | Propagates to sub-dirs | SSR | CSR |
+|-------------------|---------------------------------------------------------------------|:----------------------:|:---:|:---:|
+| +page.svelte      | Create an HTML page                                                 |           ❌            |  ✅  |  ✅  |
+| +error.svelte     | Create an HTML page to render HTTP errors                           |           ✅            |  ✅  |  ✅  |
+| +layout.svelte    | Create an HTML layout                                               |           ✅            |  ✅  |  ✅  |
+| +page.js          | Provide data for the **page** via the `load()` function             |           ❌            |  ✅  |  ✅  |
+| +page.server.js   | API endpoint for this **page**, automatically called at page load   |           ❌            |  ✅  |  ❌  |
+| +server.js        | API endpoint for this **page**, to be manually called via `fetch()` |           ❌            |  ✅  |  ❌  |
```

Comment:
This isn't just for a page. It can be called from an external service, etc.
```suggestion
| +server.js        | API endpoint                                                     |           ❌            |  ✅  |  ❌  |
```